### DISCUSSION TITLE
Is there has a dev or test network for create smaller Plot?

### DISCUSSION THREAD

**Discussion by EysaDun** [Chia-Network/chia-blockchain#4543] [Q&A]
overview source code of Chia, we need modify some code for ploting. So, creating a smaller Plot could reduce the test cycle time for developing. It will cost 6-20 hours for one Plot, it's so Expensive.
Who can tell me how to join dev or test network and creating a timeless Plot?

**Comment by 7by24** [comment]
you cat try it on testnet,

**Reply by EysaDun** [reply]
Has any doc for how to connect to testnet?

**Reply by 7by24** [reply]
FYI
[https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#testnets](url)

**Reply by EysaDun** [reply]
OK，i view the page for testnets. And i have another Question is：
Is the testnet only support run chia by cli? it seems GUI can not run on testnet.
